apps/codelab/src/app/components/exercise/exercise.component.html

```diff
@@ -24,6 +25,10 @@
         [bootstrap]="bootstrapTest"
         [code]="code"
       ></codelab-babel-test-runner>
+      <div *ngIf="isSolved" class="solved-message">
+        <h3>Go to the next slide</h3>
+        <img src="/assets/images/well_done.gif" alt="solved-message" />
```

Comment:
Please add a more human readable alt, and add i18n-alt attribute for accessibility and

Reply:
@kirjs Is 'Tests passed successfully and you can go to the next slide' sounds good? May I use it?

Reply:
I'd just say something like "Well done! Now you can go to the next slide"